Cineaste/Extension/NSManagedObjectContext+Save.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NSManagedObjectContext+Save.swift
+//  Cineaste App
+//
+//  Created by Felizia Bernutz on 29.12.18.
+//  Copyright © 2018 spacepandas.de. All rights reserved.
+//
+
+import CoreData
+
+extension NSManagedObjectContext {
+    func saveOrRollback(completion: ((_ result: Result<Bool>) -> Void)? = nil) {
+        guard hasChanges else {
```

Comment:
This is needed to not trigger a change to update the UI when using pull-to-refresh and there even is no update of any movie.

Reply:
On what devices did we have those weird display issues again?

Pull-To-Refresh looks perfectly fine on my iPhone 7.

Maybe it was a simulator-only issue?

Reply:
It is only an issue when there are new movie data. Like described in my (general) comment, we can simulate this situation with changing some data in CoreData. I tested it with a real device (iPhone 7 Plus) and I can still reproduce it. There is only a smooth pull to refresh animation when there are no data updates.

Reply:
Can you try on your device with the latest version of this branch? I think I solved it...

Reply:
Now it looks good on my device 🤩